crates/bevy_ecs/src/query/state.rs

```diff
@@ -827,6 +834,9 @@ impl<Q: WorldQuery, F: WorldQuery> QueryState<Q, F> {
     /// the current change tick are given. This is faster than the equivalent
     /// iter() method, but cannot be chained like a normal [`Iterator`].
     ///
+    /// # Panics
+    /// [`ComputeTaskPool`] is not stored as a resource in `world`.
```

Comment:
These docs should probably be updated to mention that the panic happens if the compute task pool wasn't present at initialization, and that said compute pool is guaranteed to be present if they've accessed it inside a system, since the scheduler initializes the ComputeTaskPool.

Reply:
Done!